metaflow/plugins/metadata/service.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, environment, flow, event_logger, monitor):
     def compute_info(cls, val):
         v = val.rstrip("/")
         try:
-            resp = requests.get(os.path.join(v, "ping"), headers=SERVICE_HEADERS)
+            resp = requests.get(os.path.join(v, "ping"), headers=SERVICE_HEADERS.copy())
```

Comment:
multiple instances of `.copy()` in this file. Same thoughts, if this is related to the Otel feature or remains of some other fix?

Reply:
Same as above

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1729